ddcz/templates/creations/rating.html

```diff
@@ -1,5 +1,9 @@
 <span class="rating-stars">
 {% for i in rating %}
-    <img src="{{ skin_rating_star_url }}"{% if forloop.first %} alt="{{ rating_description }}"{% endif %}>
+    {% if i == 5 %}
+        <img src="{{ dragon_url }}" alt="Vítěz ZD" class="dragon">
```

Comment:
Unrelated, but for the future: I'd put in the full "Vitěz O Hlavu Zlatého Draka" since alt is mostly used for text-to-speech readers